paragon-cli.js

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
+
+const inquirer = require('inquirer');
```

Comment:
Oooh neat, I was looking at `inquirer` awhile back. Seems pretty cool, and more powerful than the `commander` package we've previously been using to build CLI tools.